tests/unit/processor/ip_informer/test_ip_informer.py

```diff
@@ -183,6 +184,46 @@
             },
         },
     ),
+    (
+        "single field with ipv4 address and filtered properties",
+        {
+            "filter": "ip",
+            "ip_informer": {
+                "source_fields": ["ip"],
+                "target_field": "result",
+                "properties": ["is_loopback"],
+            },
+        },
+        {"ip": "192.168.5.1"},
+        {
+            "ip": "192.168.5.1",
+            "result": {
+                "192.168.5.1": {
+                    "is_loopback": False,
```

Comment:
would it make more sense to test for a property that is != False in order to differentiate from the second test case?

Reply:
this testcase tests that it should be possible to only extract a single property. 
the second testscase tests, that it should be possible to extract a properte which does not exist for a ipv4address.
the value of the fields does not matter for the first testcase.

Reply:
fair enough.